examples/js/loaders/GLTFLoader.js

```diff
@@ -765,7 +765,8 @@ THREE.GLTFLoader = ( function () {
 					fog: true,
 					lights: true,
 					opacity: params.opacity,
-					transparent: params.transparent
+					transparent: params.transparent,
+					depthWrite: params.depthWrite
```

Comment:
Nit: rather than passing in an arg, can you just do what` LDrawLoader` does and set
```js
material.depthWrite = ! material.isTransparent;
```
Personal preference only.